lib/IRGen/LoadableByAddress.cpp

```diff
@@ -942,11 +942,25 @@ static SILInstruction *createOutlinedCopyCall(SILBuilder &copyBuilder,
   return copy;
 }
 
+// Find the correct debug scope for alloc stack. We want to give to the
+// expanded sequence the correct debug scope so we skip over instructions
+// that aren't lowered to anything real (e.g. debug_value).
+static const SILDebugScope *findAllocStackDebugScope(SILBasicBlock &BB) {
```

Comment:
I think this utility would make sense in SILBasicBlock.h or even as a member function of SILBasicBlock. It's going to be useful for other passes that have similar issues, too. Perhaps call it `getScopeOfFirstNonMetaInstruction`.